addons/project/static/src/js/project_calendar.js

```diff
@@ -0,0 +1,15 @@
+odoo.define('project.ProjectCalendarController', function (require) {
+"use strict";
+const CalendarController = require('web.CalendarController');
+
+
+const ProjectCalendarController = CalendarController.include({
```

Comment:
Include ? Why not an extend, with a proper js_class ?

Reply:
That's better indeed. Done